core/snippets/snippets/genericForLoopStatement.snippet

```diff
@@ -0,0 +1,11 @@
+name: genericForLoopStatement
```

Comment:
Not sure why this isn't just a for loop snippet. Do we really need a new generic for loop snippet?